.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: ''
+assignees: guzongmin
+
+---
+
+<!--- Provide a general summary of the issue in the Title above -->
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. 
+2. 
+3. 
+4. 
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Logs**
+If applicable, add logs to help explain your problem.
+
+**Environment (please complete the following information):**
```

Comment:
I think `(please complete the following information)` should be removed.

Reply:
It is used to highlight that the below information is important.

Reply:
Maybe it needs some rewording. The sentence looks redundant to me. I think every section requires the issue submitter to complete some information, doesn't it? Why need to point that out for the environment section in particular?